### PR TITLE
Redirect donors 2

### PR DESCRIPTION
**Pull request by pablo-code-org**
- This splits the library for pegasus and dashboard so they consume the data from different data sources.

## PR Checklist:

- [x] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: lib/cdo/graphics/certificate_image.rb
Comment (davidsbailey) [inline lib/cdo/graphics/certificate_image.rb]: hey @pablo-code-org , the challenge here is that certificate_image.rb is used by both pegasus and dashboard. Due to some quirks in the way we run pegasus locally and in drone, I have some concerns about this particular change sneaking through drone but then breaking in the DTT. If you want to proceed with this, I would suggest spinning up an adhoc and making sure the hour_of_code_finish.feature UI test is passing there.

alternately, what we had talked about at the end of last week was putting this change on hold until the congrats work is done, which I am planning to complete in the next few weeks.

Comment (pablo-code-org) [reply lib/cdo/graphics/certificate_image.rb]: Oh, I'll leave the PR ready and wait for the image certificate changes

**Comment by cat5inthecradle** [comment]
@pablo-code-org @davidsbailey Did the blocking work get completed and is this ready to be picked back up?

**Comment by davidsbailey** [comment]
@cat5inthecradle no, the blocking work is still in progress, currently actively being worked on.

**Comment by davidsbailey** [comment]
I got eager and merged latest staging (including the blocking congrats changes) into this PR 😝 after resolving merge conflicts and changing one `PegasusCdoDonors` to `DashboardCdoDonors` I think it's good to go! @sureshc would you mind also reviewing since now pablo and I have both contributed commits to this PR?

**Review by davidsbailey — APPROVED** [review]

**Pull request merged**
